content/documentation/distributed-tracing/index.adoc

```diff
@@ -20,51 +20,40 @@ toc::[]
 
 Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
 
-icon:bullhorn[size=2x]{nbsp} To embed Jaeger, you must have the same protocol in Kiali and Tracing service.
+icon:bullhorn[size=2x]{nbsp} To have the tracing integration, you must set the configuration in external services / tracing.
 
-== Kiali and Tracing Service have the same protocol
+== Tracing url set
 
-When Kiali and Tracing service have the same protocol, we'll have the embedded feature in the website.
+When Tracing url is set, Kiali show a link to the Distributed Tracing in the menu.
 
-In the Distributed Tracing link in the menu we can see a full view for search the traces of our services.
+image::menu_external_link.png[Distributed Tracing View]
 
-image::distributed_tracing_same_protocol.png[Distributed Tracing View]
+== Tracing in_cluster_url set
 
+In each service we'll have a new tab to display the traces for the service in a scatter plot. (If we have error traces this tab will display them)
```

Comment:
Once the `in_cluster_url` setting is set on the [Kiali CR](https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml#L448), each service page will have the new Traces tab where a scatter plot will be displayed with its related traces. 

I would avoid the usage of `we`.